boards/arm/nucleo_l011k4/pinmux.c

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2020 Steven Daglish <s.c.daglish@gmail.com>
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <kernel.h>
+#include <device.h>
+#include <init.h>
+#include <drivers/pinmux.h>
+#include <sys/sys_io.h>
+
+#include <pinmux/stm32/pinmux_stm32.h>
+
+/* pin assignments for NUCLEO-L011K4 board */
+static const struct pin_config pinconf[] = {
+#if DT_NODE_HAS_STATUS(DT_NODELABEL(i2c1), okay) && CONFIG_I2C
+	{STM32_PIN_PA4, STM32L0_PINMUX_FUNC_PA4_I2C1_SCL},
+	{STM32_PIN_PA10, STM32L0_PINMUX_FUNC_PA10_I2C1_SDA},
+#endif
```

Comment:
This could be moved to device tree as well

Reply:
Moved from pinmux.c to device tree.